derive-macros/src/lib.rs

```diff
@@ -124,9 +124,9 @@ fn gen_schema_fields(data: &Data) -> TokenStream {
                             ).to_compile_error()
                         }
                     }
-                    quote_spanned! { field.span() => #(#type_path_quoted),* get_nullable_container_struct_field(stringify!(#name))}
```

Comment:
Wow, subtle! So a path like `std::collections::HashMap<String, String>` was being emitted as `std::,collections::,HashMap<String, String>`?

Reply:
yep

Reply:
how did this compile before then haha?

Reply:
I think it was never used with fully qualified types (or partially qualified types that contained more that one part separated by ::). `#(),*` acts kinda like `','.join` in Python - no comma is inserted if there is only one element in the list

Reply:
Actually there is another fix few lines above I forgot to add, will do that soon